blog/blog-content.php

```diff
@@ -0,0 +1,26 @@
+<?php include '../includes/header.php';?>
+
+<div class="hero-image full-width bottom-spacer"></div> <!-- End hero image -->
+<div class="content-divider bottom-spacer"></div>
+<!-- End of content-divider -->
+
+<div class="header-text-component bottom-spacer">
```

Comment:
All this content will be dynamically added through js, you won't need it.

Reply:
okay